client/blocks/all-sites/index.jsx

```diff
@@ -72,16 +72,16 @@ class AllSites extends Component {
 		return (
 			<div className={ allSitesClass }>
 				<a
-					className="site__content"
+					className="all-sites__content"
 					href={ href }
 					onMouseEnter={ this.props.onMouseEnter }
 					onMouseLeave={ this.props.onMouseLeave }
 					onClick={ this.onSelect }
 				>
 					{ showCount && this.renderSiteCount() }
-					<div className="site__info">
-						<span className="site__title">{ title || translate( 'All My Sites' ) }</span>
-						{ domain && <span className="site__domain">{ domain }</span> }
+					<div className="all-sites__info">
```

Comment:
It looks like we lost some flex properties specific to `.site__info` and `.site__content` (above) with this change. Those styles are in `/blocks/site/style.scss`.

<img width="601" alt="screen shot 2018-04-17 at 11 46 56" src="https://user-images.githubusercontent.com/448298/38881360-a3522790-4235-11e8-80f2-524c3a4aa5ee.png">